samples/correlation-csharp/FunctionAppCorrelation/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "IsEncrypted": false,
```

Comment:
nit: the indentation seems to be a mix of TABs and SPCs, also considering an empty line before EOF.

Reply:
I"m creating different branch for the full samples. (I didn't want to have a big pr). 
I'll fix this on the branch! Thank you!